### PR TITLE
Feature/form hook: Edit Profile

### PR DESCRIPTION
**Pull request by ryansnorek**
## Description

Implement `useForms` hook in `EditProfile` component

[Video](https://www.loom.com/share/f6f9a1ff162e4569849a48b8b5cf0590)

Fixes # ([Trello](https://trello.com/c/rkXEUmPG))

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Profile/EditProfile.js [hunk 1]
```diff
@@ -2,51 +2,39 @@ import React, { useState } from 'react';
...
+  const showModal = () => setModalOpen(true);
 
-  const showModal = () => {
-    setModalOpen(true);
-  };
-
-  const handleCancel = () => {
-    setModalOpen(false);
-  };
+  const handleCancel = () => setModalOpen(false);
```
Comment (MrJacobSullivan) [inline src/components/pages/Profile/EditProfile.js:24]: I definitely like this change. Anytime we can reduce three lines into one is a plus!

**Review by MrJacobSullivan — APPROVED** [review]
This is fantastic. The `useForm` hook you created is making these files much more succinct.

File: src/components/pages/Profile/EditProfile.js [hunk 2]
```diff
@@ -2,51 +2,39 @@ import React, { useState } from 'react';
...
 
-  const handleOk = () => {
+  const handleSubmit = e => {
     setModalOpen(false);
+    console.log(formValues);
```
[the remark this answers is not in the document]
Comment (ryansnorek) [reply src/components/pages/Profile/EditProfile.js:28]: I'm not seeing this console log on the branch anymore for some reason

File: src/components/pages/Profile/EditProfile.js [hunk 3]
```diff
@@ -2,51 +2,39 @@ import React, { useState } from 'react';
 import { Form, Input, Button, Radio, Modal, TreeSelect } from 'antd';
 import '../../../styles/styles.css';
 import { connect } from 'react-redux';
+import useForms from '../../../hooks/useForms';
+
+const initialValues = {
```
Comment (Alex-Hoskins) [inline src/components/pages/Profile/EditProfile.js:7]: Is it necessary for these initialValues to have data within them? Or could they be an empty string?

**Review by Alex-Hoskins — APPROVED** [review]
Code looks good. Switching to the useForms hook was a good call. Overall clean code and thorough video. Nice work!

File: src/components/pages/Profile/EditProfile.js (hunk 3 above)
Comment (ryansnorek) [reply src/components/pages/Profile/EditProfile.js:7]: These temporary initial values need to be replaced by user profile data. I think we can pull it from the redux store but that would be for another ticket

**Review by Fvaldenegro95 — APPROVED** [review]
Looks great! Going to look even better once the redux store is in fully.

**Review by danielaescutia07 — APPROVED** [review]
Great implementation and readable code is always appreciated! Good job!

**Comment by r-gio28** [comment]
awesome switch to the useForms hook. very clear loom walkthrough as well!

**Pull request merged**